django-cms/django-cms
# add jQuery to local namespace in order to initialize cancel.js

## Description

#6026 has been closed without being fixed. This pull request fixes that issue.

How to reproduce:
* Edit a CMS page and go into structure mode.
* Open Developer Tools in the browser.
* In Structure Mode, click onto **Delete** to delete a plugin.
* A popup window appears. The developer tools complain: `Uncaught ReferenceError: django is not defined`.
* Therefore the button "No, take me back" located inside the frame, doesn't work anymore.

This is because `cancel.js` attempts to access `django.jQuery` which is not yet added to the global JavaScript namespace.
And the reason for the latter is, that the Django View method
`cms.admin.placeholderadmin.PlaceholderAdminMixin.delete_plugin()`
does not define any media in its context.

## Checklist

* [x] I have opened this pull request against ``release/3.7.x``
* [x] I have updated the **CHANGELOG.rst**
* [o] I have added or modified the tests when changing logic
